src/map/skill.c

```diff
@@ -9541,11 +9541,16 @@ static int skill_castend_nodamage_id(struct block_list *src, struct block_list *
 				                    skill->castend_nodamage_id);
 			}
 			break;
-		case RK_IGNITIONBREAK:
+				case RK_IGNITIONBREAK:
 		case LG_EARTHDRIVE:
 			{
 				int splash;
+#if PACKETVER >= 20180207
```

Comment:
if you not totally sure about packet version in all 3 supported clients, better add comment, ``unconfirmed``